build.gradle

```diff
@@ -18,8 +18,10 @@ buildscript {
         maven { url "http://kotlin.bintray.com/kotlin-dev" }
         maven { url "https://plugins.gradle.org/m2/" }
         maven { url 'https://maven.fabric.io/public' }
+        maven { url "http://storage.googleapis.com/r8-releases/raw/master" }
     }
     dependencies {
+        classpath Dep.GradlePlugin.r8
```

Comment:
Must be before the Gradle Plugin for Android in the following conversation.So I wrote here.
https://issuetracker.google.com/issues/114554345#comment17